deployment/keycloak/local-test-users.json

```diff
@@ -0,0 +1,49 @@
+[ {
```

Comment:
Is this for development and testing purposes only?

Reply:
jepp